website/docs/quickstart_guide/docker.md

```diff
@@ -41,13 +43,25 @@ They are not suitable for most production use-cases because they keep all data i
    `ferretdb` runs FerretDB.
 
 2. Start services with `docker compose up -d`.
+   To be sure that you've fetched the latest version you can precede the command with `docker compose pull`.
 
 3. If you have `mongosh` installed, just run it to connect to FerretDB.
-   If not, run the following command to run `mongosh` inside the temporary MongoDB container, attaching to the same Docker network:
+   It will use credentials passed in `mongosh` flags or MongoDB URI
+   to authenticate to the PostgreSQL database.
+   You'll also need to set `authMechanism` to `PLAIN`.
+   The example URI would look like:
 
    ```sh
-   docker run --rm -it --network=ferretdb --entrypoint=mongosh mongo mongodb://ferretdb/
```

Comment:
Let's remove `sh` in \`\`\`sh. It's not a bash :hugs:

Reply:
Well, that's a shell command with an URL as an argument.

@noisersup CI fails now

Reply:
It's not a shell command, but github view makes it to look like one :)

I've sent the commit to quickly and didn't cover everything yet. It will be ready in a moment